.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,110 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Integration Tests
```

Comment:
This is too generic a name. We have several actions that are running integration tests.
The `pr-integration` workflow and the `nightly-tests` workflow both run them.
Let's come up with something more specific, like `release-promotion-tests`

 Does any process run this on `dev`? It is not part of our regular testing in WSM eng.

Reply:
This [workspacemanager-integration-test-trigger](https://fc-jenkins.dsp-techops.broadinstitute.org/job/workspacemanager-integration-test-trigger/) jenkins job is kicked off after WSM is deployed to dev. Although worth noting that QA/release train mostly looks at only alpha and staging runs.

Reply:
Just for my information, can you explain who runs that jenkins job on dev? Right now when we merge to the dev branch, the merge script writes to a repo and argo deploys that version to dev. I didn't think there was a tie in to Jenkins.

Reply:
Sure. The `gke-deploy` job that apparently deploys WSM (and other apps) resides in Jenkins [here](https://fc-jenkins.dsp-techops.broadinstitute.org/job/gke-deploy/). DevOps can speak better to if/how `gke-deploy` is tied to argo, but there's a shell script in that job that runs the project's docker image.

After deploy, the `gke-deploy` then triggers WSM tests as part of release promotion.

I believe there are plans to move the `gke-deploy` job to GitHub actions in the future, though.